### ISSUE TITLE
[Packaging] [Debian] debconf stuff broken?

### ISSUE THREAD

**Issue by mcdope** [mcdope/pam_usb#169]
## BLOCKS #165 

While testing #165 I've noticed that the debconf scripts seem to be broken, at least on Ubuntu 22.04.

Output of install after purge: 
```
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'libpam-usb' instead of './libpam-usb_0.8.3-RC1_amd64.deb'
The following NEW packages will be installed:
  libpam-usb
0 upgraded, 1 newly installed, 0 to remove and 3 not upgraded.
Need to get 0 B/49.0 kB of archives.
After this operation, 225 kB of additional disk space will be used.
Get:1 /home/tobias/Downloads/libpam-usb_0.8.3-RC1_amd64.deb libpam-usb amd64 0.8.3 [49.0 kB]
perl: warning: Setting locale failed.
perl: warning: Please check that your locale settings:
	LANGUAGE = (unset),
	LC_ALL = "c",
	LANG = "de_DE.UTF-8"
    are supported and installed on your system.
perl: warning: Falling back to a fallback locale ("de_DE.UTF-8").
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
Vorkonfiguration der Pakete ...
Error looking up object for device /dev/0x03000000
ls: cannot access '/dev/0x03000000*': No such file or directory
Usage:
  udisksctl info [OPTION?]

Show information about an object.

Options:
  -p, --object-path      Object to get information about
  -b, --block-device     Block device to get information about
  -d, --drive            Drive to get information about

/usr/bin/locale: Cannot set LC_CTYPE to default locale: No such file or directory
/usr/bin/locale: Cannot set LC_MESSAGES to default locale: No such file or directory
/usr/bin/locale: Cannot set LC_ALL to default locale: No such file or directory
Selecting previously unselected package libpam-usb.
(Reading database ... 261249 files and directories currently installed.)
Preparing to unpack .../libpam-usb_0.8.3-RC1_amd64.deb ...
Unpacking libpam-usb (0.8.3) ...
Setting up libpam-usb (0.8.3) ...
locale: Cannot set LC_CTYPE to default locale: No such file or directory
locale: Cannot set LC_MESSAGES to default locale: No such file or directory
locale: Cannot set LC_ALL to default locale: No such file or directory
Error looking up object for device /dev/0x03000000
ls: cannot access '/dev/0x03000000*': No such file or directory
Usage:
  udisksctl info [OPTION?]

Show information about an object.

Options:
  -p, --object-path      Object to get information about
  -b, --block-device     Block device to get information about
  -d, --drive            Drive to get information about

/usr/bin/locale: Cannot set LC_CTYPE to default locale: No such file or directory
/usr/bin/locale: Cannot set LC_MESSAGES to default locale: No such file or directory
/usr/bin/locale: Cannot set LC_ALL to default locale: No such file or directory
Created symlink /etc/systemd/system/multi-user.target.wants/pamusb-agent.service → /lib/systemd/system/pamusb-agent.se
rvice.
Processing triggers for man-db (2.10.2-1) ...
```

**Comment by mcdope** [comment]
lol?! interestingly the config is generated just fine...

**Comment by mcdope** [comment]
seems like `debian/config` line 12 doesn't return anymore what we expect.

**Comment by mcdope** [comment]
Looks like it picks up the serial of mmcblk0 as device too. Maybe because it has no revision / is a cheapo device?

**Comment by mcdope** [comment]
Indeed I can't repro this on my desktop, with the same Ubuntu version using a brand usb stick.

Seems to be a quirk handling devices with missing attributes.

**Comment by mcdope** [comment]
Should be fixed in https://www.dropbox.com/s/vsadwvzg0pj6tv8/libpam-usb_0.8.3-RC2_amd64.deb?dl=0

**Comment by mcdope** [comment]
Yep, fixed by 7c5e976
